perf/rt_tests.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Would you please add license heading? If the test is inspired please do include the original author along with link to that file.